src/app/cic/controllers/checkDetails.js

```diff
@@ -16,11 +16,12 @@ class CheckDetailsController extends DateController {
       const changeUrl = req.sessionModel.get("changeUrl");
       const firstName = req.sessionModel.get("firstName");
       const middleName = req.sessionModel.get("middleName");
-      const surname = req.sessionModel.get("surname");
-      const fullName = firstName + " " + middleName + " " + surname;
-
+      const surname = req.sessionModel.get("surname")
+      const fullName = firstName + " " + middleName + " " + surname
+      const language = req.lng
```

Comment:
Looks like we're missing some ; here. Zac has merged a PR with linter scripts, might want to give those a run?